tests/lib/rules/no-array-prototype-extensions.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-array-prototype-extensions');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('@babel/eslint-parser'),
+  parserOptions: { ecmaVersion: 2020, sourceType: 'module' },
+});
+
+ruleTester.run('no-array-prototype-extensions', rule, {
+  valid: [
```

Comment:
Can you add these as valid test cases:
* `firstObject`
* `lastObject`

Reply:
yes! good points!